packages/cli/src/cli/basemaps/action.import.ts

```diff
@@ -0,0 +1,224 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { Epsg } from '@basemaps/geo';
+import {
+    Aws,
+    Const,
+    DefaultBackground,
+    LogConfig,
+    RecordPrefix,
+    TileMetadataImageryRecord,
+    TileMetadataRecord,
+    TileMetadataTable,
+    TileMetadataTag,
+} from '@basemaps/shared';
+import { CommandLineAction, CommandLineFlagParameter, CommandLineStringParameter } from '@rushstack/ts-command-line';
+import assert from 'assert';
+import { DynamoDB } from 'aws-sdk';
+import { promises as fs } from 'fs';
+import { CogStacJob } from '../../cog/cog.stac.job';
+import { createImageryRecordFromJob } from '../cogify/action.batch';
+import { TagChanges, TileSetUpdater } from './tileset.updater';
+import { invalidateXYZCache, rgbaToHex, showDiff } from './tileset.util';
+
+async function showUpdateTag(
+    tag: TileMetadataTag,
+    changes: TagChanges,
+    imagery: Map<string, TileMetadataImageryRecord>,
+    isCommit: boolean,
+): Promise<string> {
+    let output = `\nChanges for ${tag} ${changes.name}/${changes.projection.toEpsgString()}:\n`;
+    const backgroundBefore = rgbaToHex(changes.before.background ?? DefaultBackground);
+    const backgroundAfter = rgbaToHex(changes.after.background ?? DefaultBackground);
+    if (backgroundBefore !== backgroundAfter) {
+        output += `\nBackground changed from '${backgroundBefore}' to '${backgroundAfter}'\n`;
+    }
+
+    output += showDiff(changes.after, changes.before, imagery);
+    await invalidateXYZCache(changes.name, changes.projection, tag, isCommit);
+    return output;
+}
+
+async function clearDB(table: TileMetadataTable): Promise<void> {
```

Comment:
This seems like a super dangerous function to have, we should never be deleting from the database.

Reply:
I have moved it into the `loadFull` function but this is needed to populate dev with production or to restore production if something goes wrong. This is ported code from the basemaps-team repository

Reply:
I disagree, we should never be wiping a database with a script, this has too many ways of going wrong.

Can we just create a new head record and insert it? I don't see why we would ever need to delete the old records?

Reply:
Okay I'll remove the full option